packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1838,7 +1838,7 @@ export const components = ({
           },
         }),
         input: {
-          height: "auto",
+          height: `calc(${odysseyTokens.Spacing7} - (${odysseyTokens.BorderWidthMain} * 2))`,
```

Comment:
I still don't understand why we need to set input height explicitly. Seems like content + padding should just be the correct height right?

Reply:
Naw. Shred made a comment about how this works, and noted we needed to keep it fixed.

Reply:
Yes, except for the acrobatics we need to subtract out the border size, which adds another 2px to the overall height. We _could_ make this implicit, but it would involve a lot of tweaking of line-heights and such. I'd rather leave this as a fixed value for now until we (someday, maybe) do a full refactor.

Reply:
I guess what I mean is, this was always `height: auto` up until ~2 weeks ago and everything was seemingly fine.